### DISCUSSION TITLE
Is there a demo about multisig on bitcion?

### DISCUSSION THREAD

**Discussion by jiuyaopinyin** [trustwallet/wallet-core#2343] [Q&A]
Are there any demos about Bitcoin multi-signature that I can refer to? Please!
@hewigovens @catenocrypt

**Comment by hewigovens** [comment]
No demo for multi sig

**Reply by jiuyaopinyin** [reply]
> No demo for multi sig

😂,Is the current WalletCore version capable of multi-signature?

I found this method in the source file 
   https://github.com/trustwallet/wallet-core/blob/5c2dbd2019f2215b40d1fcaa1767bcfe5459a742/src/Bitcoin/Script.cpp#L119
But there is no such method in the swift file, what is the reason?

**Reply by hewigovens** [reply]
It's not exposed, I don't think you can use MultiSig from swift / kotlin now

**Reply by jiuyaopinyin** [reply]
> It's not exposed, I don't think you can use MultiSig from swift / kotlin now

OK,Thanks,And is there a plan about the MultiSig?
